src/positions/saga.ts

```diff
@@ -21,9 +23,11 @@ const POSITIONS_FETCH_TIMEOUT = 45_000 // 45 seconds
 
 async function fetchPositions(walletAddress: string) {
   const response = await fetchWithTimeout(
-    // TODO: Use the final API URL once it's ready
-    'https://plugins-api-oaxbpxoaha-uc.a.run.app/balances?' +
-      new URLSearchParams({ network: 'celo', address: walletAddress }),
+    `${networkConfig.cloudFunctionsUrl}/hooks-api/getPositions?` +
```

Comment:
The existing pattern/convention is to explicitly define URLs in `src/web3/networkConfig` (for better or worse).  Can you do that instead (e.g., add a properties to the `networkConfigs`)?  And separately we can discuss changing the convention?

Reply:
Oh no, I've wanted to refactor that for ages. And I've already broken that [convention](https://github.com/valora-inc/wallet/blob/76971cf0cba23e8261553bcc10f30e923097426a/src/exchange/CeloNewsFeed.tsx#L45) 😅